platform/broadcom/docker-syncd-brcm-rpc.mk

```diff
@@ -17,7 +17,7 @@ SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_SYNCD_BRCM_RPC)
 endif
 
 $(DOCKER_SYNCD_BRCM_RPC)_CONTAINER_NAME = syncd
-$(DOCKER_SYNCD_BRCM_RPC)_VERSION = 1.0.0-rpc
+$(DOCKER_SYNCD_BRCM_RPC)_VERSION = 1.0.0+rpc
```

Comment:
Yes, there is a desing for version string, we are following https://semver.org/. This was a flaw of initial implementation. I chose to put rpc with a dash, which in semver means that this is a pre-release version, which it is not. It causes an extension installation failure that requires syncd>=1.0.0 but according to semver rules 1.0.0-rpc is less then 1.0.0. With this change I put rpc sign as a build metadata part which solves an issue because 1.0.0+rpc satisfies the constraint >=1.0.0.

Reply:
Debian does not follow semver. They are defined by semver rules and there are online tools to check https://semvercompare.azurewebsites.net/?version=1.0.0&version=1.0.0-rpc&version=1.0.0%252Brpc.

Reply:
Why `1.0.0-rpc` < `1.0.0` < `1.0.0+rpc` ? Is it just string comparison?